lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/client/AzureBatchRuntimeConfiguration.java

```diff
@@ -58,17 +58,59 @@
    */
   public static final RequiredParameter<String> AZURE_BATCH_POOL_ID = new RequiredParameter<>();
 
+  /**
+   * The environment variable that holds the path to the default configuration file.
+   */
+  public static final String AZBATCH_CONFIGURATION_FILE_ENVIRONMENT_VARIABLE = "REEF_AZBATCH_CONF";
+
   /**
    * The ConfigurationModule for the local resourcemanager.
    */
   public static final ConfigurationModule CONF = new AzureBatchRuntimeConfiguration()
-      .merge(CommonRuntimeConfiguration.CONF)
+      .merge(AzureBatchRuntimeConfigurationStatic.CONF)
       .bindNamedParameter(AzureBatchAccountUri.class, AZURE_BATCH_ACCOUNT_URI)
       .bindNamedParameter(AzureBatchAccountName.class, AZURE_BATCH_ACCOUNT_NAME)
       .bindNamedParameter(AzureBatchAccountKey.class, AZURE_BATCH_ACCOUNT_KEY)
       .bindNamedParameter(AzureBatchPoolId.class, AZURE_BATCH_POOL_ID)
-      // Bind the Azure Batch runtime
-      .bindImplementation(JobSubmissionHandler.class, AzureBatchJobSubmissionHandler.class)
-      .bindImplementation(DriverConfigurationProvider.class, AzureBatchDriverConfigurationProvider.class)
       .build();
+
+  /**
+   * Returns a HDInsight runtime configuration from the credentials stored in the given file.
```

Comment:
Let's fix this comment to replace HDInsight references with Azure Batch. Also applies to line 81.

Reply:
fixed.